daisy_workflows/windows_upgrade/cleanup.wf.json

```diff
@@ -0,0 +1,42 @@
+{
+  "Name": "cleanup",
+  "DefaultTimeout": "5m",
```

Comment:
Might be too short in some cases.

Reply:
All 3 operations takes only <2s usually. No time consuming step.